acceptance/openstack/workflow/v2/workflows_test.go

```diff
@@ -8,12 +8,27 @@ import (
 	th "github.com/gophercloud/gophercloud/testhelper"
 )
 
-func TestWorkflowsCreate(t *testing.T) {
+func TestWorkflowsCreateDelete(t *testing.T) {
 	client, err := clients.NewWorkflowV2Client()
 	th.AssertNoErr(t, err)
 
 	workflow, err := CreateWorkflow(t, client)
 	th.AssertNoErr(t, err)
+	defer DeleteWorkflow(t, client, workflow)
 
 	tools.PrintResource(t, workflow)
 }
+
+func TestWorkflowsGet(t *testing.T) {
```

Comment:
You can probably just add the Get call within the `TestWorkflowsCreateDelete` test. It'll save some duplicated testing.